providers/implementations/keymgmt/ec_kmgmt.c

```diff
@@ -170,21 +167,19 @@ int key_to_params(const EC_KEY *eckey, OSSL_PARAM_BLD *tmpl, int include_private
         if (ecbits <= 0)
             goto err;
         sz = (ecbits + 7 ) / 8;
-        if (!ossl_param_bld_push_BN_pad(tmpl,
-                                        OSSL_PKEY_PARAM_PRIV_KEY,
-                                        priv_key, sz))
+        if (!ossl_param_build_set_bn_pad(tmpl, params,
+                                         OSSL_PKEY_PARAM_PRIV_KEY, priv_key,
```

Comment:
Nit: I actually find it nicer if `OSSL_PKEY_PARAM_PRIV_KEY` stands alone on its line, just as in the other calls.

Reply:
updated